src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15380/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15380/project/issues?id=sample&branch=feature1&issues=AWEyf-3Q3ebqF5v7XZH9&open=AWEyf-3Q3ebqF5v7XZH9)

[ISSUE_KEY]: AWEyf-3Q3ebqF5v7XZH9